### PR TITLE
Fix HTML iframe tag in EmbedController(s) and adopt relative URLs

### PR DESCRIPTION
**Pull request by valerio-bozzolan**
This changeset fixes this issue:
* https://github.com/LibreTime/libretime/issues/988

This changeset contributes in fixing this issue
* https://github.com/LibreTime/libretime/issues/957

### CODE REVIEW

**Comment by paddatrapper** [comment]
This PR doesn't include a fix for the quoting of the iframe: 

```
<iframe id="player_iframe" frameborder="0" width="100%" style="bottom:0px; left:0px; position:fixed; right:0px;" src=http://localhost:8080/embed/player?stream=auto&style=premium></iframe>
```

**Comment by paddatrapper** [comment]
The other thing to consider here is that the embeddable widgets, like the schedule, can be embedded into sites that aren't the LibreTime instance. How are relative URLs handled within iframes? Also `getStationUrl()` obeys 'force_ssl` in the config file, so that can be used to ensure it uses SSL even if the config has a different port configured

**Comment by valerio-bozzolan** [comment]
> This PR doesn't include a fix for the quoting of the iframe:
> 
> ```
> <iframe id="player_iframe" frameborder="0" width="100%" style="bottom:0px; left:0px; position:fixed; right:0px;" src=http://localhost:8080/embed/player?stream=auto&style=premium></iframe>
> ```

Uhm. This also fixes the double-quotes in the iframe's `src=""`. Probably it does not fix what do you mean and I've missed something. What do you mean instead?

**Comment by valerio-bozzolan** [comment]
Edited: I missed something. I will git commit --force to avoid two commits.

File: airtime_mvc/application/views/scripts/index/index.phtml [hunk 1]
```diff
@@ -57,7 +57,7 @@
         <div id="tab-4" class="podcast tab_content">
             <div class="rss-title-wrapper">
                 <div id="rss-subscribe">
-                    <a target="_blank" href="<?php echo $this->stationPodcastRssUrl ?>" class="rss_icon"><span><?php echo _("Subscribe") ?></span><img src="css/radio-page/img/rss.png"></a>
+                    <a target="_blank" href="<?php echo $this->stationPodcastRssUrl ?>" class="rss_icon"><span>Subscribe </span><img src="css/radio-page/img/rss.png"></a>
```
Comment (paddatrapper) [inline airtime_mvc/application/views/scripts/index/index.phtml:60]: Why did you remove the translation call here?

File: airtime_mvc/application/views/scripts/index/index.phtml [hunk 2]
```diff
@@ -70,7 +70,7 @@
             <div id="jp_container_1" class="jp-audio">
                 <div class="jp-type-playlist">
 
-                    <div class="current-track track-overflow"><?php echo _("Now Playing:") ?><span class="track-overflow"></span></div>
+                    <div class="current-track track-overflow">Now Playing: <span class="track-overflow"></span></div>
```
Comment (paddatrapper) [inline airtime_mvc/application/views/scripts/index/index.phtml:73]: And the translation call here?

File: airtime_mvc/application/views/scripts/index/index.phtml (hunk 1 above)
Comment (paddatrapper) [inline airtime_mvc/application/views/scripts/index/index.phtml:60]: Why did you remove the translation call here?

File: airtime_mvc/application/views/scripts/index/index.phtml (hunk 2 above)
Comment (paddatrapper) [inline airtime_mvc/application/views/scripts/index/index.phtml:73]: And the translation call here?

File: airtime_mvc/application/views/scripts/index/index.phtml [hunk 3]
```diff
@@ -355,4 +355,14 @@
 
     });
 
-</script>
+</script><link href="/css/radio-page/custom.css" rel="stylesheet" type="text/css" />
```
Comment (paddatrapper) [inline airtime_mvc/application/views/scripts/index/index.phtml:358]: Did you mean to add this? Considering it is Google Analytics, please remove it from the PR

**Review by paddatrapper — CHANGES_REQUESTED** [review]

**Comment by valerio-bozzolan** [comment]
The problem is I'm editing directly from your production Docker container to see if everything works (`/usr/share/airtime`, not versioned) and committing in top of your master assuming they were the same version.

I will fix ASAP.

**Comment by valerio-bozzolan** [comment]
OK I'm fine filling GitHub with trash.
Now this commit is OK from my perspective.

Thanks for your review!

File: airtime_mvc/application/views/scripts/index/index.phtml (hunk 1 above)
Comment (valerio-bozzolan) [reply airtime_mvc/application/views/scripts/index/index.phtml]: Unintended.

Fixed in https://github.com/LibreTime/libretime/pull/1119/commits/b8f4a3392716b5be2ab89139cc5b65b973dec523.

File: airtime_mvc/application/views/scripts/index/index.phtml (hunk 2 above)
Comment (valerio-bozzolan) [reply airtime_mvc/application/views/scripts/index/index.phtml]: Unintended.

Fixed in https://github.com/LibreTime/libretime/pull/1119/commits/b8f4a3392716b5be2ab89139cc5b65b973dec523.

File: airtime_mvc/application/views/scripts/index/index.phtml (hunk 3 above)
Comment (valerio-bozzolan) [reply airtime_mvc/application/views/scripts/index/index.phtml]: Unintended.

Fixed in https://github.com/LibreTime/libretime/pull/1119/commits/b8f4a3392716b5be2ab89139cc5b65b973dec523.

**Comment by paddatrapper** [comment]
LGTM, thanks!

**Comment by hairmare** [comment]
Thanks for cleaning up this ❤️

proto+host relative URLs make much more sense.

In the long run we'd also like to make it easy to be hosted on subpath without using a base tag but that's gonna be part of the refactor to vue & a drf based api first approach!

**Comment by valerio-bozzolan** [comment]
> Thanks @valerio-bozzolan for cleaning up this heart
> 
> proto+host relative URLs make much more sense.
> 
> In the long run we'd also like to make it easy to be hosted on subpath.

Good luck.
